x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/use_response_actions_log_table.tsx

```diff
@@ -177,13 +179,28 @@ export const useResponseActionsLogTable = ({
             description: (
               // codeblock for output
               <StyledEuiCodeBlock data-test-subj={getTestId('details-tray-output')}>
-                {isExpired
-                  ? OUTPUT_MESSAGES.hasExpired(command)
-                  : isCompleted
-                  ? wasSuccessful
-                    ? OUTPUT_MESSAGES.wasSuccessful(command)
-                    : OUTPUT_MESSAGES.hasFailed(command)
-                  : OUTPUT_MESSAGES.isPending(command)}
+                {isExpired ? (
+                  OUTPUT_MESSAGES.hasExpired(command)
+                ) : isCompleted ? (
+                  wasSuccessful ? (
+                    isGetFileCommand ? (
```

Comment:
Could you rewrite this without having a lot of nested conditions?

Reply:
Yeah we can do better. I'll refractor this so it's easier to reason with.

Reply:
be87496f92c26d57837903dc6eafaed4668830b1